packages/generate-arg-types/package.json

```diff
@@ -6,6 +6,8 @@
   "homepage": "https://webstudio.is",
   "type": "module",
   "scripts": {
+    "dev": "build-package --watch",
+    "build": "build-package",
```

Comment:
This is cli only tool. We run it with tsx directly. Why do you want to build it?

Reply:
Got it, you reuse types. We will need to rename it to something like arg-types then. But maybe later.

Reply:
It now exports some code as well